akd/src/tests.rs

```diff
@@ -110,10 +110,6 @@ fn setup_mocked_db(db: &mut MockLocalDatabase, test_db: &AsyncInMemoryDatabase)
             futures::executor::block_on(tmp_db.batch_get::<TreeNodeWithPreviousValue>(key))
         });
 
-    let tmp_db = test_db.clone();
-    db.expect_batch_get::<Azks>()
-        .returning(move |key| futures::executor::block_on(tmp_db.batch_get::<Azks>(key)));
-
```

Comment:
Hmm, any reason for removing this?

Reply:
<img width="1368" alt="Screenshot 2024-06-21 at 11 27 24 AM" src="https://github.com/facebook/akd/assets/13973013/3353657b-a3bd-43e2-8e8f-b75cc49ffb33">
It appears to be a dupe/typo, and from my local test-runs has no bearing on test outcomes